en/creating-an-artboard.md

```diff
@@ -66,7 +66,7 @@ You may have noticed that in the image above the libraries also appear to be pre
     This example is for an iPhone 11 in portrait orientation (414x896). If you don't see this preset artboard, choose Apple Devices from the dropdown at the top of the panel.
 
   <div class="divider--half"></div>
-  <img class="responsive-img" src="images/getting-started-artboard.png" />
+  <img class="responsive-img" src="images/getting-started-artboard.png" srcset="images/getting-started-artboard@2x.png" />
```

Comment:
This image is also huge, we need to downsize it to e.g. 70% so that it is more or less quite similar to the images that follow.